src/columns.jl

```diff
@@ -1015,6 +1018,24 @@ t     x  y
 julia> t == t2
 false
 
+```
+
+If `col` is not an existing column, `setcol` will add it:
+
+```jldoctest setcol
+julia> t = table([1,2], [2,3], names = [:a,:b])
+Table with 2 rows, 2 columns:
+a  b
+────
+1  2
+2  3
+
+julia> setcol(t, :c, [1,2])
+Table with 2 rows, 3 columns:
+a  b  c
+───────
+1  2  1
+2  3  2
```

Comment:
Is there a `setcol(t, cols::Pair...)`?

Reply:
Surprisingly it was documented already but not implemented :) So it does not show the documentation in the diff as it was there already and I just left it. But yes, `setcol(t, cols::Pair...)` works now, see [this test](https://github.com/JuliaComputing/IndexedTables.jl/pull/161/files#diff-1649e312fc01ff2d3b8d62d9a4e334c5R546).

Reply:
LOL